### PR TITLE
✨ Create Header Component

### PR DESCRIPTION
**Pull request by mrana17**
- Main Header
- styled
- add header background color to `global.css`

### CODE REVIEW

**Review by DNaunin — APPROVED** [review]
LGTM! 🚀

**Review by mewiemers — COMMENTED** [review]
take a ride! 🏍GJ!!

**Review by cboenke — APPROVED** [review]
GJ! 🚀

File: components/header/header.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import styles from "./header.module.css";
+
+function Header() {
+  return <section className={`${styles.header}`}></section>;
```
Comment (m-wiemers) [inline components/header/header.tsx]:
```suggestion
  return <section className={styles.header}></section>;
```

**Review by m-wiemers — APPROVED** [review]
This is really unbelievably brilliant. Never read anything better!

**Review by RonjaMeinecke — APPROVED** [review]
💥GJ

**Review by phlgr — COMMENTED** [review]
Gj 🚀

**Pull request merged**